pkg/agent/datamodel/gpu_components.go

```diff
@@ -116,6 +116,10 @@ var FabricManagerGPUSizes = map[string]bool{
 	"standard_nd92isr_h100_v5":  true,
 	"standard_nd96isr_h100_v5":  true,
 	"standard_nd100isr_h100_v5": true,
+	// H200
```

Comment:
Should we add all of these to this list? 
```
Standard_ND96isr_H200_v5
Standard_ND96isrf_H200_v5
Standard_ND96is_H200_v5
standard_nd96isr_h200_v5
Standard_ND96isf_H200_v5
Standard_NDC96is_H200_v5
```

Reply:
All except 1 (Standard_NDC96is_H200_v5) are upper/lower case variants. Agentbaker code makes it into lower case:
```func GPUNeedsFabricManager(size string) bool {
	return datamodel.FabricManagerGPUSizes[strings.ToLower(size)]
}```

Reply:
`Standard_NDC96is_H200_v5` is the only additional one but I don't see any usage or reference to it in any docs.